### PR TITLE
Transfer money scenario

### PR DESCRIPTION
**Pull request by anthonyflea**
Transfer money scenario.
Please do a code review.

### CODE REVIEW

File: src/main/java/pl/edu/agh/iet/katabank/Account.java [hunk 1]
```diff
@@ -60,4 +60,12 @@ private void checkValidAmount(BigDecimal amount) {
...
     }
+
+    public void transfer(Account targetAccount, BigDecimal bigDecimal) {
+        if (targetAccount == null) {
+            throw new RuntimeException("Target account cannot be null.");
+        }
+        this.withdraw(bigDecimal);
+        targetAccount.deposit(bigDecimal);
+    }
 }
```
Comment (aaalata) [inline src/main/java/pl/edu/agh/iet/katabank/Account.java:71]: Hi. We have the method tested with cucumber - with correct inputs. Please unit test with targetAccount being null.

**Review by aaalata — CHANGES_REQUESTED** [review]
Good job. Just a test missing. Thanks!

**Comment by aaalata** [comment]
Good job. Just a test missing. Thanks!

**Comment by anthonyflea** [comment]
I've added new test.
By the way, I've found and corrected some errors.

**Pull request merged**